css/style.css

```diff
@@ -57,6 +56,99 @@ body {
   display: none;
 }
 
+.mobile-menu {
+  display: none;
+  position: absolute;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  width: 425px;
+  height: 812px;
+  background: #fff;
+  border-radius: 0 0 50px 50px;
+  box-shadow: 0 48px 48px rgb(37, 47, 137, 0.08);
+}
```

Comment:
[OPTIONAL] You did a great job with your mobile menu, well done 😄! Please take into account that most of the mobile menus stick to the user's screen. This means that, when scrolling down, the menu is still displayed at the top of the page. Currently, your menu is static at the beginning of your page, causing this effect when scrolling down: 
![image](https://user-images.githubusercontent.com/42799579/201748512-60025515-a892-48b8-918c-d8f9b65982fc.png)

So, to improve the UX/UI of your webpage, kindly consider this advice 😄. 
> You could do it by giving a position: fixed to your menu 😉.